quickstarts/releaseiq/config.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the quickstart (required)
+name: releaseiq
+
+# Displayed in the UI (required)
+title: ReleaseIQ
+
+# Long-form description of the quickstart (required)
+description: |
+  ## What's ReleaseIQ?
+  ReleaseIQ is the Unified Enterprise DevOps Platform to Accelerate Software Delivery. ReleaseIQ is focused on solving the problems of multi-tool, team, and product release pipelines.
+
+  For ReleaseIQ intergration with NewRELIC, you must have :
+  - New Relic License to be configured in ReleaseIQ Platform.
+  - Configure NewRelic in ReleaseIQ platform Observability tool settings.
+
+  ReleaseIQ Features with NewRelic tool
+  - Custom Events : During pipeline execution for each step, events with all step information is sent to NewRelic. To enable this feature, need to configure notification to NewRelic in each step.
+  - Deployment Verification : Deployments can be verified by NewRelic based on the newrelic policies configured in deploy step of a pipeline and based on the priority alerts will be sent back to ReleaseIQ platform. Progress of Pipeline to subsequent steps will be based on deploymnet verififcation result.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
```

Comment:
At a minimum for this to be a quickstart you need a docs url reference. I don't see that in the configuration. Please review our contributor guide for more information on adding a documentation URL.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#documentation